styles/globals.css

```diff
@@ -216,3 +216,7 @@ svg:has(> #Mail) {
 [data-tracked-repo-wizard] > button:focus-within > [data-button-title] {
   @apply text-green-800;
 }
+
+.workspace-drop-down > [role="combobox"] {
```

Comment:
The `<SingleSelect />` uses a Radix select primitive that generate a button with a role of combobox, so need to do some custom CSS.